dist/styles.css

```diff
@@ -1,5 +1,72 @@
-div {
-  height: 50vh;
-  width: 50vw;
-  background-color: red;
+@media (prefers-color-scheme: no-preference) {
+  :root {
+    --bg-base: #fff;
+    --font-color: #222;
+  }
+}
+
+@media (prefers-color-scheme: light) {
+  :root {
+    --bg-base: #fff;
+    --font-color: #222;
+  }
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --bg-base: #0e1216;
+    --font-color: #aaa;
+  }
+}
+
```

Comment:
I like these media queries.  Haven't used prefers-color-scheme before myself but it looks good!